cfep-18.md

```diff
@@ -0,0 +1,74 @@
+
+<table>
+<tr><td> Title </td><td> Uploading binaries from pull requests </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Chris Burr &lt;christopher.burr@cern.ch&gt;</td></tr>
+<tr><td> Created </td><td> June 06, 2020</td></tr>
+<tr><td> Updated </td><td> June 06, 2020</td></tr>
+<tr><td> Discussion </td><td> NA </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Allow package binaries to be uploaded from builds in pull requests.
+
+## Motivation
+
+Sometimes it would be useful to access binaries built in pull requests. Reasons include:
+
+* Testing GUI applications
+* More easily testing the effects on a pull requests on downstream packages. 
+  For example, patches in llvm affecting builds of clang.
```

Comment:
Shouldn't this be a `test_downstreams`?

Reply:
Ideally yes but it's sometimes just impractical with the current infrastructure (like in this case of llvmdev+clangdev or clangdev+root/cling).